cg/apps/tb/models.py

```diff
@@ -0,0 +1,45 @@
+from pydantic import BaseModel, validator
+from typing import Optional
+from dateutil.parser import parse as parse_datestr
+from pathlib import Path
+import datetime as dt
+
+
+class TrailblazerAnalysis(BaseModel):
+    id: int
+    family: str
+    case_id = Optional[str]
+
+    version: Optional[str]
+    logged_at: Optional[str]
+    started_at: Optional[str]
+    completed_at: Optional[str]
+    status: Optional[str]
+    priority: Optional[str]
```

Comment:
These values may or may not be set or returned by Trailblazer, but I thought it would be good to declare all possible fields that can be returned by the API